base/src/main/java/edu/iu/IuText.java

```diff
@@ -38,6 +38,8 @@
  * Low-level text processing utilities.
  */
 public final class IuText {
+	
+	private static final Base64.Encoder URL_ENCODER = Base64.getUrlEncoder().withoutPadding();
```

Comment:
Why do we need this constant for URL_ENCODER, but not for URL_DECODER?

Reply:
Based on JDK 21 source, getUrlEncoder() and getUrlDecoder() refer to constants already so no need to duplicate those. However, withoutPadding() always creates a new Encoder instance so I opted to create a constant to avoid unnecessary object creation.